imsim/readout.py

```diff
@@ -270,16 +269,31 @@ def get_primary_hdu(eimage, lsst_num, camera_name=None,
         telcode = 'CC'
     else:
         phdu.header['FILTER'] = LSSTCam_filter_map.get(band, None)
-        phdu.header['INSTRUME'] = 'LSSTCam'
+        if camera_name == 'LsstCamSim':
+            phdu.header['INSTRUME'] = 'LSSTCamSim'
+        else:
+            phdu.header['INSTRUME'] = 'LSSTCam'
         phdu.header['RAFTBAY'] = raft
         phdu.header['CCDSLOT'] = sensor
         phdu.header['RA'] = ratel
         phdu.header['DEC'] = dectel
         phdu.header['ROTCOORD'] = 'sky'
+        phdu.header['ROTPA'] = rotang
         telcode = 'MC'
     dayobs = eimage.header['DAYOBS']
     seqnum = eimage.header['SEQNUM']
     contrllr = eimage.header['CONTRLLR']
+    phdu.header['TELESCOP'] = SIMONYI_TELESCOPE
+    phdu.header['TELCODE'] = telcode
+    phdu.header['RASTART'] = ratel
+    phdu.header['DECSTART'] = dectel
+    phdu.header['ELSTART'] = eimage.header['ALTITUDE']
+    phdu.header['AZSTART'] = eimage.header['AZIMUTH']
+    if eimage.header['IMGTYPE'] == 'SKYEXP':
+        phdu.header['RADESYS'] = 'ICRS'
+        phdu.header['TRACKSYS'] = 'RADEC'
+    else:
```

Comment:
When does this get used?  Simulated flats?

Reply:
(by this, I mean TRACKSYS = LOCAL).

Reply:
Yes, for anything not on-sky, specifically any calibration frames.